src/core/Microsoft.PowerToys.Settings.UI/Strings/de-de/Resources.resw

```diff
@@ -0,0 +1,593 @@
+  </data>
+  <data name="AppDescription" xml:space="preserve">
+    <value>Microsoft.PowerToys.Settings.UI</value>
+    <comment>Application description</comment>
+  </data>
+  <data name="Shell_Main.Content" xml:space="preserve">
+    <value>Hauptseite</value>
+    <comment>Navigation view item name for Main</comment>
+  </data>
+  <data name="Shell_General.Content" xml:space="preserve">
+    <value>Allgemein</value>
+    <comment>Navigation view item name for General</comment>
+  </data>
+  <data name="Shell_PowerLauncher.Content" xml:space="preserve">
+    <value>PowerToys Run</value>
+    <comment>Navigation view item name for PowerToys Run</comment>
+  </data>
+  <data name="Shell_PowerRename.Content" xml:space="preserve">
+    <value>PowerRename</value>
+    <comment>Navigation view item name for PowerRename</comment>
+  </data>
+  <data name="Shell_ShortcutGuide.Content" xml:space="preserve">
+    <value>Shortcut Guide</value>
+    <comment>Navigation view item name for Shortcut Guide</comment>
+  </data>
+  <data name="Shell_PowerPreview.Content" xml:space="preserve">
+    <value>File Explorer Preview</value>
+    <comment>Navigation view item name for File Explorer Preview</comment>
+  </data>
+  <data name="Shell_FancyZones.Content" xml:space="preserve">
+    <value>FancyZones</value>
+    <comment>Navigation view item name for FancyZones</comment>
+  </data>
+  <data name="Shell_ImageResizer.Content" xml:space="preserve">
+    <value>Image Resizer</value>
+    <comment>Navigation view item name for Image Resizer</comment>
+  </data>
+  <data name="Shell_KeyboardManager.Content" xml:space="preserve">
+    <value>Keyboard Manager</value>
+    <comment>Navigation view item name for Keyboard Manager</comment>
+  </data>
+  <data name="KeyboardManager_ConfigHeader.Text" xml:space="preserve">
+    <value>Aktuelle Konfiguration</value>
+    <comment>Keyboard Manager current configuration header</comment>
+  </data>
+  <data name="KeyboardManager_Description.Text" xml:space="preserve">
+    <value>Rekonfigurieren Sie Ihre Tastatur durch Neuzuordnung von Tasten und Tastenkombinationen</value>
+    <comment>Keyboard Manager page description</comment>
+  </data>
+  <data name="KeyboardManager_EnableToggle.Header" xml:space="preserve">
+    <value>Tastatur-Manager aktivieren</value>
+    <comment>Keyboard Manager enable toggle header</comment>
+  </data>
+  <data name="KeyboardManager_ProfileDescription.Text" xml:space="preserve">
+    <value>Wählen Sie das Profil aus, um die aktive Tastenzuordnung und Tastenkürzel anzuzeigen</value>
+    <comment>Keyboard Manager configuration dropdown decription</comment>
+  </data>
+  <data name="KeyboardManager_RemapKeyboardButton.Content" xml:space="preserve">
+    <value>Eine Taste neu zuordnen</value>
+    <comment>Keyboard Manager remap keyboard button content</comment>
+  </data>
+  <data name="KeyboardManager_RemapKeyboardHeader.Text" xml:space="preserve">
+    <value>Tastatur neu zuordnen</value>
```

Comment:
Please change to "Taste neu zuordnen" because we remap a single key and not the whole keyboard! This is also false in the english version!